src/ui/public/angular-bootstrap/datepicker/year.html

```diff
@@ -1,15 +1,72 @@
-<table role="grid" aria-labelledby="{{uniqueId}}-title" aria-activedescendant="{{activeDateId}}">
+<table
```

Comment:
the selected range is not the most obvious here.. is there a reason not to highlight the range with the same background as the bounding dates?